src/gui/resources/translations/de.ts

```diff
@@ -0,0 +1,2749 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="de_DE">
+<context>
+    <name>AnimAddDialog</name>
+    <message>
+        <location filename="../../animadddialog.ui" line="20"/>
+        <source>Add Animation</source>
+        <translation>Animation hinzufügen</translation>
+    </message>
+    <message>
+        <location filename="../../animadddialog.ui" line="75"/>
+        <source>Preset:</source>
+        <translation>Vorlage:</translation>
+    </message>
+    <message>
+        <location filename="../../animadddialog.ui" line="104"/>
+        <source>Animation:</source>
+        <translation>Animation:</translation>
+    </message>
+    <message>
+        <location filename="../../animadddialog.ui" line="153"/>
+        <source>Show preview</source>
+        <translation>Vorschau anzeigen</translation>
+    </message>
+</context>
+<context>
+    <name>AnimDetailsDialog</name>
+    <message>
+        <location filename="../../animdetailsdialog.ui" line="20"/>
+        <source>Loaded Animations</source>
+        <translation>Geladene Animationen</translation>
+    </message>
+    <message>
+        <location filename="../../animdetailsdialog.ui" line="48"/>
+        <source>Name</source>
+        <translation></translation>
```

Comment:
There are 2 files as I started with de_DE.ts then noticed that not all desktop environments allow multiple languages, so German (Austria) for example would just fall back to English. But we can just remove de_DE.ts and use de.ts and there's no difference.

I started by modifying the file manually, and this leave-out way was used in the en_GB.ts. Then I switched to using _Qt Linguist_, because it shows a preview of the dialog where unmodified strings are displayed as #String, which just bothered me. I don't think there's a benefit, though.

![Qt Linguist shows untranslated strings with an initial #](https://user-images.githubusercontent.com/30179385/128493140-e2a40509-934e-4678-9430-bbf4538c8e11.jpg)

Reply:
Explicit translation to `Name` would be preferred. I have done so as well for the greek translation and I think it makes more sense.